api/src/main/java/org/openmrs/module/allergyapi/Allergen.java

```diff
@@ -76,6 +76,26 @@ public void setCodedAllergen(Concept codedAllergen) {
 			nonCodedAllergen = null;
 		}
     }
+    
+	/**
+	 * Sets other non coded concept uuid constant.
+	 * 
+	 * @param otherNonCodedConceptUuid
+	 * @since 2.0
+	 */
+    public static void setOtherNonCodedConceptUuid(String otherNonCodedConceptUuid) {
+    	OTHER_NON_CODED_UUID = otherNonCodedConceptUuid;
+    }
+    
+	/**
+	 * Returns other non coded concept uuid constant.
+	 * 
+	 * @return other non coded concept uuid constant
+	 * @since 2.0
```

Comment:
The since attribute should have the version of module instead of platform. Does this make sense?